python/tchannel/thrift/scheme.py

```diff
@@ -28,7 +28,7 @@
 from .. import scheme
 
 
-class ThriftArgScheme(scheme.ArgScheme):
+class ThriftArgScheme(scheme.RawArgScheme):
```

Comment:
But I think it is also make sense. You can image the rawargscheme is the raw base scheme. Any json or thrift arg scheme is a variation of raw arg scheme becuase raw arg scheme basically does nothing.